elasticsearch.yml.tpl

```diff
@@ -21,4 +21,6 @@ discovery:
   zen:
     hosts_provider: gce
 
+cluster.initial_master_nodes: [${nodes}]
```

Comment:
Jak se tohle zachová po restartu celého clusteru? Jakože všechny node down a všechny up? Nečetl jsem k tomu docku, takže popravdě furt moc nechápu jak tento parametr funguje, ale zdá se mi divné že by tam člověk jenom vylistoval nody a tím bylo hotovo.

Reply:
Bohužel bez toho to nenahodíš :-| háže to:
```
[2020-04-07T15:19:51,343][WARN ][o.e.c.c.ClusterFormationFailureHelper] [elasticsearch-prod-0] master not discovered yet, this node has not previously joined a bootstrapped (v7+) cluster, and [cluster.initial_master_nodes] is empty on this node: have discovered [{elasticsearch-prod-0}{nlgXLwvcSnaLPLDc4B4nhA}{VA4M1ZAxTwCuKnwiU5Pp2w}{10.156.0.58}{10.156.0.58:9300}{dilm}{ml.machine_memory=3869970432, xpack.installed=true, ml.max_open_jobs=20}, {elasticsearch-prod-2}{XIYmNokfRe-vaFMXdio92A}{2tDSSoQ-Sk-txf6ebKoAQQ}{10.156.0.57}{10.156.0.57:9300}{dilm}{ml.machine_memory=3869970432, ml.max_open_jobs=20, xpack.installed=true}, {elasticsearch-prod-1}{BXFtnvvPRsmI4QSpvXI_lw}{mCCpj-yuQQelspKXzHQsEw}{10.156.0.59}{10.156.0.59:9300}{dilm}{ml.machine_memory=3869986816, ml.max_open_jobs=20, xpack.installed=true}]; discovery will continue using [127.0.0.1:9301, 127.0.0.1:9302, 127.0.0.1:9303, 127.0.0.1:9304, 127.0.0.1:9305, [::1]:9301, [::1]:9302, [::1]:9303, [::1]:9304, [::1]:9305, 10.156.0.59:9300, 10.156.0.57:9300, 10.156.15.233:9300] from hosts providers and [{elasticsearch-prod-0}{nlgXLwvcSnaLPLDc4B4nhA}{VA4M1ZAxTwCuKnwiU5Pp2w}{10.156.0.58}{10.156.0.58:9300}{dilm}{ml.machine_memory=3869970432, xpack.installed=true, ml.max_open_jobs=20}] from last-known cluster state; node term 0, last-accepted version 0 in term 0
```
Na to nastavení jsem přišel googlením a taky moc nevím, jak se to chová. Můžeme to samozřejmě otestovat, jestli chceš.